environment.yml

```diff
@@ -10,9 +10,12 @@ dependencies:
     - bokeh
     - datashader
     - geopandas
+    - gdal=2.1.*
```

Comment:
We may not need to pin this. It was pinned because of ongoing annoyances between conda-forge and defaults which may have been resolved by now.